src/pages/index.js

```diff
@@ -231,20 +232,21 @@ function ZoneView({ provinceId, zoneNo }) {
                 position: "relative",
               }}
             >
-              <Arrow />
-              เขตเลือกตั้งที่ {zone.no}
-              <div
-                css={{
-                  display: "inline-block",
-                  border: "solid #212121",
-                  borderWidth: "0 2px 2px 0",
-                  padding: "4px",
-                  transform: "rotate(-45deg)",
-                  verticalAlign: "middle",
-                  marginLeft: "12px",
-                  cursor: "pointer",
+              <Arrow
+                onLeftArrowClick={() => {
+                  navigate(zonePath({ provinceId, no: zoneNo - 1 || 1 }))
                 }}
-              />
+                onRightArrowClick={() =>
+                  navigate(
+                    zonePath({
+                      provinceId,
+                      no: zoneNo < province.zone ? zoneNo + 1 : province.zone,
```

Comment:
For these two boundary cases, what do you think if we hide the corresponding arrow or do a circular visit?

Reply:
@heytitle  I think it would be nice if we hide the corresponding arrow, I will hide it.

Reply:
thanks!